src/Radiance_Monitor/nwprod/gdas_radmon/driver/test_jgdas_verfrad_hera.sh

```diff
@@ -40,27 +40,30 @@ export global_shared_ver=v14.1.0
 #############################################################
 
 export RADMON_SUFFIX=${RADMON_SUFFIX:-testrad}
-export NWTEST=${NWTEST:-/scratch1/NCEPDEV/da/Edward.Safford/noscrub/GSI/util/Radiance_Monitor/nwprod}
+export NWTEST=${NWTEST:-/scratch1/NCEPDEV/da/Edward.Safford/noscrub/GSI-monitor/src/Radiance_Monitor/nwprod}
```

Comment:
Is it too hard-wired to replace with `$USER`.  It should have the desired effect.

Reply:
Can do.